src/ResourceManager/Compute/Commands.Compute.Test/Common/ComputeTestController.cs

```diff
@@ -96,6 +97,10 @@ public void RunPsTest(params string[] scripts)
             string callingClassType,
             string mockName)
         {
+            Dictionary<string, string> d = new Dictionary<string, string>();
```

Comment:
This is used only for a few of the test projects which depend on authorization RP. Not needed for all tests. Is there benefit in moving it there? Keeping it here also makes it explicit that en exclusion is added for particular test projects only.